spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/it/DatastoreIntegrationTests.java

```diff
@@ -845,15 +850,34 @@ public void testPageableGqlEntityProjectionsSlice() {
 		assertThat(testEntityProjections.get(0).getColor()).isEqualTo("blue");
 	}
 
-	//An index containing both "color" and "size" is required
-	@Ignore
-	@Test
-	public void testSliceString() {
-		Slice<String> slice =
-				this.testEntityRepository.getSliceStringBySize(2L, PageRequest.of(0, 3));
+	@Test(timeout = 10000L)
+	public void testSliceString() throws IOException, URISyntaxException, InterruptedException {
+		URL resource = this.getClass().getResource("/index.yaml");
```

Comment:
Can you put the index creating logic in a helper function and then put it in the `@Before` of the test? So that in the future if we have another test that needs an index it will be clear how to add it.